packages/vuetify/src/components/VDialog/VDialog.ts

```diff
@@ -241,68 +241,55 @@ export default baseMixins.extend({
   },
 
   render (h): VNode {
-    const children = []
-    const data = {
-      class: this.classes,
-      ref: 'dialog',
-      directives: [
-        {
-          name: 'click-outside',
-          value: this.onClickOutside,
-          args: {
-            closeConditional: this.closeConditional,
-            include: this.getOpenDependentElements,
+    let dialog = this.showLazyContent(() => {
```

Comment:
I feel like this should live in its own method. it makes the render fn feel overly verbose.

Reply:
I wasn't trying to do a full rewrite or anything, these functions hadn't really changed at all.